src/edgekv/ekv-cli-main.ts

```diff
@@ -141,7 +141,7 @@ del
 
 const list = program.command('list')
   .alias("l")
-  .description("List all the namespaces or list all items with in a group. Use list -h to see available options");
+  .description("List all the namespaces or List the data groups for a given namespace in an Akamai environment. Use list -h to see available options");
```

Comment:
```suggestion
  .description("List the namespaces or the data groups for a given namespace in an Akamai environment. Use list -h to see available options");
```

Reply:
It may actually be good to confirm the whole statement with Sarah; my suggestion is mostly to make the sentence less clumsy and remove the double capitalization of "List" in one sentence.